apps/site/pages/ltb/entry/[slug]/index.tsx

```diff
@@ -0,0 +1,1041 @@
+		);
+	};
+
+	useEffect(() => {
+		window.history.pushState({}, "", "/ltb/entry/" + (currentIndex + 1));
+	}, [currentIndex]);
+
+	return (
+		<div className="flex justify-center">
+			<div className="w-2/3 py-4">
+				<div className="mb-4 flex justify-center">
+					<EntrySwitcher
+						maxLength={entries.length}
+						setIndex={setCurrentIndex}
+						currentIndex={currentIndex}
+					/>
+				</div>
+				<LearningDiaryEntryForm
+					entry={entries[currentIndex]}
+					learningDiaryInformation={learningDiaryInformation}
+					onUpdate={handleUpdateEntry}
+				/>
+			</div>
+		</div>
+	);
+}
+
+function EntrySwitcher({
+	maxLength,
+	setIndex,
+	currentIndex
+}: {
+	maxLength: number;
+	setIndex: (index: number) => void;
+	currentIndex: number;
+}) {
+	const handlePrev = () => {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-expect-error
+		setIndex(prevIndex => Math.max(prevIndex - 1, 0));
+	};
+
+	const handleNext = () => {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-expect-error
+		setIndex(prevIndex => Math.min(prevIndex + 1, maxLength - 1));
+	};
+
+	return (
+		<div className="flex space-x-16">
+			<button className="btn btn-primary" onClick={handlePrev} disabled={currentIndex === 0}>
+				Vorheriger Eintrag
+			</button>
+			<div className="rounded bg-gray-100 p-2">{currentIndex + 1}</div>
+			<button
+				className="btn btn-primary"
+				onClick={handleNext}
+				disabled={currentIndex === maxLength - 1}
+			>
+				Nächster Eintrag
+			</button>
+		</div>
+	);
+}
+
+function LearningDiaryEntryForm({
+	entry,
+	learningDiaryInformation,
+	onUpdate
+}: {
+	entry: LearningDiaryEntryResult;
+	learningDiaryInformation: LearningDiaryInformation;
+	onUpdate: (updatedEntry: LearningDiaryEntryResult) => void;
+}) {
+	const methods = useForm({
+		defaultValues: {
+			learningLocation: entry.learningLocation,
+			effortLevel: entry.effortLevel,
+			distractionLevel: entry.distractionLevel,
+			notes: entry.notes,
+			learningTechniqueEvaluation: entry.learningTechniqueEvaluation
+		}
+	});
+
+	const { handleSubmit, reset } = methods;
+
+	const { mutateAsync: updateLearningDiaryEntry } = trpc.learningDiaryEntry.update.useMutation();
+
+	const onSubmit = async (data: any) => {
+		const updatedEntry: LearningDiaryEntryResult = {
+			...entry,
+			...data
+		};
+
+		await updateLearningDiaryEntry({
+			id: updatedEntry.id,
+			learningLocationId: updatedEntry.learningLocation?.id ?? "",
+			effortLevel: updatedEntry.effortLevel ?? 0,
+			distractionLevel: updatedEntry.distractionLevel ?? 0,
+			notes: updatedEntry.notes ?? ""
+		});
+
+		onUpdate(updatedEntry);
+		reset(data);
+	};
+
+	useEffect(() => {
+		reset({
+			learningLocation: entry.learningLocation,
+			effortLevel: entry.effortLevel,
+			distractionLevel: entry.distractionLevel,
+			notes: entry.notes,
+			learningTechniqueEvaluation: entry.learningTechniqueEvaluation
+		});
+	}, [entry, reset]);
```

Comment:
kann ich gerade nicht ganz überschauen, aber hier noch einmal kritisch überprüfen ob ein useEffect benötigt wird. 
Das soll eigentlich nicht der fall sein.


Beim reset evtl ? 
```suggestion
	reset({...entry});
```